search/includes/classes/class-search.php

```diff
@@ -396,6 +441,8 @@ protected function setup_hooks() {
 		if ( false !== $this->is_protected_content_enabled() ) {
 			add_filter( 'ep_indexable_post_types', array( $this, 'add_attachment_to_ep_indexable_post_types' ), 9999 );
 		}
+
+		add_filter( 'vip_search_post_meta_allow_list', array( $this, 'vip_search_post_meta_allow_list_defaults' ) );
```

Comment:
Do we want to keep with our convention of prefixing these callbacks with `filter__`?

Reply:
Changed in 7f719ce